talk/slides-llmday/custom.scss

```
@@ -0,0 +1,39 @@
+/*-- scss:defaults --*/
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)